pyproject.toml

```diff
@@ -59,6 +59,7 @@ click = "^8.1"
 Mako = "^1.2"
 ni-python-styleguide = "^0.4"
 sphinx_rtd_theme = "1.0"
+poetry-grpc-plugin = "^0.1.0"
```

Comment:
poetry-grpc-plugin requires poetry >=1.2, but the build-system section below says this package requires poetry >= 1.1.12. I have been using 1.1.15 because most internal NI components haven't upgraded past that yet.

Reply:
Do we need to specifically resolve this by mentioning the expected version number?